src/components/Forms/NewEvent.js

```diff
@@ -168,7 +168,7 @@ export default function NewEventForm(props) {
                     <Grid item xs={12}>
                         <Typography>Some common event locations (optional):</Typography>
                         <Select fullWidth onClick={handleLocation.bind(null)}>
-                            <MenuItem value={'Nest'}>Nest</MenuItem>
+                            <MenuItem key={'Nest'} value={'Nest'}>Nest</MenuItem>
```

Comment:
how come we need key/only one item has key?

Reply:
![image](https://user-images.githubusercontent.com/45786074/85801164-23adee00-b6f7-11ea-88a9-0fbf0b0d28b3.png)
 i was trynna get rid of these warnings by trying name/id/key but had no luck - followed the link in the warning and got kinda stumped too. I'll remove the key - i just accidentally left it there

Reply:
ah right. I fixed it in my pr